src/AWS.Deploy.Orchestration/RecommendationEngine/MSProjectSdkAttributeTest.cs

```diff
@@ -15,7 +15,16 @@ public class MSProjectSdkAttributeTest : BaseRecommendationTest
 
         public override Task<bool> Execute(RecommendationTestInput input)
         {
-            var result = string.Equals(input.ProjectDefinition.SdkType, input.Test.Condition.Value, StringComparison.InvariantCultureIgnoreCase);
+            bool result = false;
+            if(!string.IsNullOrEmpty(input.Test.Condition.Value))
```

Comment:
Initially I was going to add support for working by just updating the SdkType test to be able to use the AllowedValues. That ended up not working because I had the issue with the `OutputType` not being set for worker. I left this in here thinking it could be useful for future recipes.